src/interfaces/IZupRouter.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.26;
+
+import {PoolToken} from "src/types/PoolToken.sol";
+
+interface IZupRouter {
+  /**
+   * @notice emitted when the user successfully deposits into a liquidity pool
+   * @param lpTokenId the ID of the LP NFT
+   * @param token0 the token0 of the pool
+   * @param token1 the token1 of the pool
+   * @param user the user who deposited into the Pool
+   */
```

Comment:
MIssing NatSpec for the params `token0Amount`, `token1Amount` and `positionManager`